ext/node/global.rs

```diff
@@ -401,7 +409,9 @@ pub fn enumerator<'s>(
 
   let context = scope.get_current_context();
   let inner = {
-    let storage = context.get_slot::<GlobalsStorage>().unwrap();
```

Comment:
This shouldn't be panicking because it should exist. Maybe something is not setup correctly with your setup?